### PR TITLE
tlf_journal_test: Try to address the deadlock

### PR DESCRIPTION
**Pull request by jzila**
cc @strib 

I'm guessing that the journal just isn't blocking, and this should address that scenario.

### CODE REVIEW

**Comment by jzila** [comment]
I think I just changed the flake from one form to another, but at least it no longer deadlocks.

**Comment by jzila** [comment]
Ready for review.

File: go/kbfs/libkbfs/tlf_journal_test.go [hunk 1]
```diff
@@ -1513,15 +1519,11 @@ func testTLFJournalConvertWhileFlushing(t *testing.T, ver kbfsmd.MetadataVer) {
...
-		select {
-		// Now finish the block put, and let the flush finish.  We should
-		// be on a local squash branch now.
-		case unpauseBlockPutCh <- struct{}{}:
-		case err := <-errCh:
-			require.NoError(t, err)
-		}
-	}
+	// Now finish the block put, and let the flush finish.  We
+	// shouldn't be on a branch anymore.
```
Comment (strib) [inline go/kbfs/libkbfs/tlf_journal_test.go]: We should be on a branch -- the local squash branch.  In fact there's a require just below that checks that.  Same thing below.

**Comment by strib** [comment]
I'm going to fix the comment myself since you're out tomorrow and we should get this in ASAP.

**Pull request merged**